test/Api.Test/SecretsManager/Controllers/ServiceAccountsControllerTests.cs

```diff
@@ -90,12 +95,70 @@ await sutProvider.GetDependency<ICreateServiceAccountCommand>().DidNotReceiveWit
 
     [Theory]
     [BitAutoData]
-    public async void CreateServiceAccount_Success(SutProvider<ServiceAccountsController> sutProvider,
+    public async void CreateServiceAccount_OrgDoesntExist_Throws(SutProvider<ServiceAccountsController> sutProvider,
```

Comment:
If you leave this up to the query/command to check (per above), you can delete this test.

Reply:
Resolved in https://github.com/bitwarden/server/pull/3093/commits/0f3beda47fb78eddc0a67108e3f6922345c93dc5